iocore/net/quic/QUICFrame.h

```diff
@@ -767,7 +770,7 @@ class QUICFrameFactory
    * This works almost the same as create() but it reuses created objects for performance.
    * If you create a frame object which has the same frame type that you created before, the object will be reset by new data.
    */
-  const QUICFrame &fast_create(const uint8_t *buf, size_t len, const QUICPacketR *packet);
+  QUICFrame &fast_create(const uint8_t *buf, size_t len, const QUICPacketR *packet);
```

Comment:
We should be able to keep this `const` by adding `const` to QLogFrameFactory and QLogFrame constructors.

Reply:
Because we can not convert QUICFrame to specified QUICXXXFrame with const flag.

Reply:
This works for me.
```
 return std::make_unique<Frame::AckFrame>(Convert<const QUICAckFrame>(frame));
```

Reply:
Thanks, will try it.